packages/core/src/js/feedback/FeedbackForm.types.ts

```diff
@@ -171,7 +171,7 @@ export interface FeedbackCallbacks {
   /**
    * Callback when a screenshot is added
    */
-  onAddScreenshot?: (attachFile: (filename: string, data: Uint8Array) => void) => void;
+  onAddScreenshot?: (attachFile: (filename: string, fileUri: string) => void) => void;
```

Comment:
Reflecting the recent changes, absolutely makes sense to use uri. 

I'm thinking that maybe we can simplify the callback to only accept `uri`. I don't think the file name is important, to satisfy the sentry API we can use static name like `feedback_screenshot`

````suggestion
  onAddScreenshot?: (attachFile: (uri: string) => void) => void;
````

Reply:
Good idea @krystofwoldrich 👍 
Updated with f14197d9a352b8af3d1784abdea53f36ddd99946